manual/openstack-sru-19.3.41.txt

```diff
@@ -102,3 +102,2427 @@ openstack floating ip delete $FIP_ID
+         10.095s cloud-init-local.service
+          3.708s cloud-config.service
+          2.919s apparmor.service
+          2.479s console-setup.service
+          2.249s dev-vda1.device
+          1.559s snapd.seeded.service
+          1.040s pollinate.service
+           999ms cloud-init.service
+           600ms cloud-final.service
+           579ms snapd.service
+           516ms lvm2-monitor.service
+           434ms accounts-daemon.service
+           295ms systemd-logind.service
+           293ms grub-common.service
+           278ms systemd-remount-fs.service
+           256ms ufw.service
+           250ms kmod-static-nodes.service
+           220ms dev-hugepages.mount
+           214ms open-iscsi.service
+           206ms irqbalance.service
+           204ms apport.service
+           203ms systemd-modules-load.service
+           202ms mdadm.service
+           188ms keyboard-setup.service
+           175ms networking.service
+           168ms lxd-containers.service
+           158ms systemd-journald.service
+           115ms ondemand.service
+           112ms polkitd.service
+           103ms dev-mqueue.mount
+           101ms resolvconf.service
+           100ms systemd-udev-trigger.service
+            97ms systemd-journal-flush.service
+            77ms rsyslog.service
+            67ms systemd-sysctl.service
+            59ms iscsid.service
+            59ms snapd.socket
+            57ms plymouth-read-write.service
+            56ms systemd-machine-id-commit.service
+            53ms sys-fs-fuse-connections.mount
+            52ms lxd.socket
+            50ms systemd-tmpfiles-setup-dev.service
+            50ms systemd-tmpfiles-setup.service
+            48ms systemd-timesyncd.service
+            41ms systemd-random-seed.service
+            38ms sys-kernel-debug.mount
+            38ms systemd-user-sessions.service
+            35ms user@1000.service
+            34ms ssh.service
+            33ms plymouth-quit.service
+            31ms systemd-update-utmp-runlevel.service
+            28ms systemd-update-utmp.service
+            24ms rc-local.service
+            23ms setvtrgb.service
+            19ms systemd-udevd.service
+            11ms plymouth-quit-wait.service
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@10.245.163.0 -- cloud-init analyze show
+-- Boot Record 01 --
+The total time elapsed since completing an event is printed after the "@" character.
+The time the event takes is printed after the "+" character.
+
+Starting stage: init-local
+|`->no cache found @00.02700s +00.00100s
+|`->no local data found from DataSourceNoCloud @00.22200s +00.08100s
+|`->no local data found from DataSourceConfigDrive @00.30300s +00.05600s
+|`->no local data found from DataSourceOpenNebula @00.35900s +00.02700s
+|`->no local data found from DataSourceDigitalOcean @00.38700s +00.02900s
+Starting stage: init-local/search-Azure
```

Comment:
This seems strange to see Azure in here. Wouldn't anything azure have been dropped due to ds-identify?

Reply:
Ohh nevermind, Xenial, which didn't adhere to ds-identify. not until bionic did we rule out other datasources officially.

Reply:
Yes, on Xenial we still check all ds.